### PR TITLE
Add translation of new problems (ko)

### PR DESCRIPTION
**Pull request by marocchino**
@nodeschool/seoul review please. :)

### CODE REVIEW

File: exercises/css2/problem.ko.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  tableContent: {
+    border: "1px solid black"
+  }
+};
+```
+
+`Todo`를 수정해, 체크박스가 체크되면 `tr`에 취소선을 넣고, 체크를 풀면 취소선을 지우도록 하세요.
+
+다 되었으면, `node program.js 3000 Milk 13:00`를 실행해, `http://localhost:3000`으로 들어가, 실제로 HTML이 출력되는 것을 확인하세요.
+체크박스를 몇번 클릭해, 바르게 동작하는지 확인하세요.
```
Comment (yous) [inline exercises/css2/problem.ko.md]: 몇번 → 몇 번

**Comment by yous** [comment]
I'm done.

**Comment by marocchino** [comment]
@tako-black LTGM :shipit:

**Comment by kohei-takata** [comment]
Thanks!

**Pull request merged**